cli/config/jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
We may need to set up something for styled-jsx for use with Jest.

Reply:
What do you mean by this?

Reply:
https://github.com/zeit/styled-jsx#styled-jsxcss-in-tests

I think @HendrikThePendric and @jenniferarnesen ran into problems using styled-jsx components with Jest and had to do some configuration. I don't remember the details. Could deal with that when we get there.

Reply:
Got it.  Tests do seem to work with `styled-jsx` at least with in-line styles - the example installed by `init` has `<style jsx>{...}</style>` and renders properly.  I pass our custom `babel` config explicitly to `babel-jest` which includes the `styled-jsx` transpiler, so maybe that's all that was needed?

Reply:
Actually looks like it's an issue with tagged template literals, which aren't working (again) anyway.  Will need to look into that shortly.